presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PartialAggregationPushDown.java

```diff
@@ -90,7 +90,7 @@ public PlanNode visitAggregation(AggregationNode node, RewriteContext<Void> cont
                 return context.defaultRewrite(node);
             }
 
-            boolean decomposable = node.isDecomposable(functionRegistry);
+            boolean decomposable = node.isDecomposable(functionRegistry) && node.getOrderBySymbols().isEmpty();
```

Comment:
When you add your test cases to `AbstractTestQueries` for `ORDER BY` in standard aggregations, like:

```
SELECT partkey, returnflag, array_agg(DISTINCT suppkey ORDER BY shipdate) FROM lineitem GROUP BY partkey, returnflag;
```
and replace it for example with:

```
SELECT partkey, returnflag, array_agg(DISTINCT suppkey ORDER BY shipdate) FROM lineitem GROUP BY ROLLUP (partkey, returnflag);
SELECT partkey, returnflag, array_agg(DISTINCT suppkey ORDER BY shipdate) FROM lineitem GROUP BY CUBE (partkey, returnflag);
SELECT partkey, returnflag, array_agg(DISTINCT suppkey ORDER BY shipdate) FROM lineitem GROUP BY GROUPING SETS ((), (partkey, returnflag));
```